### PR TITLE
Array manipulation 1

### PR DESCRIPTION
**Pull request by samlee0258**
Screenshot 2022-12-21 at 8 03 43 PM

### CODE REVIEW

File: array-manipulation-1/src/reverse.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    - create a for loop to loop through the array
+      - if the index is not strictly equal to -1
+        - push into the output array
+    - return output array
+*/
+
+function reverse(array) {
+  var reverseArr = [];
+  for (var i = array.length - 1; i >= 0; i--) {
+    if (i !== -1) {
```
Comment (codingcodymiller) [inline array-manipulation-1/src/reverse.js:14]: This conditional shouldn't be necessary

File: array-manipulation-1/src/tail.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - create a for loop to loop through the array
+    - create a conditional statement where if the index is not strictly equal to 0
+    - push the index value into an empty array
+  - return output array
+*/
+
+function tail(array) {
+  var tailArr = [];
+  for (var i = 0; i < array.length; i++) {
+    if (i !== 0) {
```
Comment (codingcodymiller) [inline array-manipulation-1/src/tail.js:14]: If you start the loop at `1`, this conditional will not be needed

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: array-manipulation-1/src/reverse.js (hunk 1 above)
Comment (samlee0258) [reply array-manipulation-1/src/reverse.js]: Screenshot 2022-12-28 at 4 33 27 PM

File: array-manipulation-1/src/tail.js (hunk 2 above)
Comment (samlee0258) [reply array-manipulation-1/src/tail.js]: Screenshot 2022-12-28 at 4 33 43 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**